docs/jwst/associations/association_reference.rst

```diff
@@ -263,36 +277,37 @@ Implementing ``dump()`` and ``load()``
 The base ``Association`` class defines the
 :meth:`~jwst.associations.association.Association.dump` and
 :meth:`~jwst.associations.association.Association.load` methods to
-serialize the data structure pointing to by the ``data`` attribute. If
+serialize the data structure pointed to by the ``data`` attribute. If
 the new rule uses the ``data`` attribute for storing the association
 information, no further overriding of these methods is necessary.
 
 However, if the new rule does not define ``data``, then these methods
 will need be overridden.
 
 Rule Registration
-=================
+-----------------
 
-In order for a rule to be used by ``generate``, the rule must be loaded
-into an ``AssociationRegistry``.  Since a rule is just a class that is
+In order for a rule to be used by :func:`~jwst.associations.generate`, the rule must be loaded
+into an :class:`~jwst.associations.registry.AssociationRegistry`.  Since a rule is just a class that is
 defined as part of a, most likely, larger module, the registry needs
 to know what classes are rules. Classes to be used as rules are marked
-with the ``RegistryMarker.rule`` decorator as follows::
+with the ``RegistryMarker.rule`` decorator as follows:
+
+.. code-block:: python
 
-  # myrules.py
-  from jwst.associations import (Association, RegistryMarker)
+    # myrules.py
+    from jwst.associations import (Association, RegistryMarker)
 
-  @RegistryMarker.rule
-  class MyRule(Association):
-      ...
+    @RegistryMarker.rule
+    class MyRule(Association):
+        ...
 
 Then, when the rule file is used to create an ``AssociationRegistry``,
 the class ``MyRule`` will be included as one of the available rules:
 
-.. doctest-skip::
```

Comment:
No reason to have doctest collect this only to skip it. Removing `>>>` would make this invisible to doctest and also more consistent with the rest of the code examples in "associations" doc.